### PR TITLE
Removed docs folder and minor updates

### PR DESCRIPTION
**Pull request by andreabrambilla**
`docs` removed since documentation is now built automatically

Also
- npm docs related commands now use folder docs_out
- the github-pages workflow now uses the npm scripts instead of explicit commands

### CODE REVIEW

File: .github/workflows/github-pages.yml [hunk 1]
```diff
@@ -13,27 +13,25 @@ jobs:
...
     - name: Build documentation # only if this was triggered from latest
       if: github.ref == 'refs/heads/latest' 
-      # TODO: replace these with `npm docs` and `npm postdocs` once you are sure everything works
       run: |
-        npx typedoc --out docs_out src
-        npx copyfiles images/* docs_out
-        npx copyfiles .nojekyll docs_out
-    
+        npm docs
+        npm postdocs
```
Comment (Brynjulf) [inline .github/workflows/github-pages.yml:30]: have you tested this? `npm docs` is not the same as `npm run docs`, the former will try to discern the location of the project's documentation and the latter will run a custom script. i'm not sure how/if github handles this distinction

Comment (andreabrambilla) [reply .github/workflows/github-pages.yml:30]: good point. I *think* I did, but this is a month old, I don't really remember. I'll try it locally first and see what happens

Comment (andreabrambilla) [reply .github/workflows/github-pages.yml:30]: you are right, I'll fix this

**Review by Brynjulf — APPROVED** [review]
LGTM

**Pull request merged**